README.md

```diff
@@ -14,26 +14,6 @@ The test runner requires 2 parameters:
 go run . ./testdata/practice/passing outdir
 ```
 
-### Docker
```

Comment:
Was there any particular reason why you removed this section? It can be very helpful to run the test runner using Docker to verify that it works correctly. Another benefit of running using Docker is that you can simulate the production settings (no network). For the C# test runner, I created a separate script: https://github.com/exercism/csharp-test-runner/blob/master/run-in-docker.ps1

I'd suggest keeping this section, albeit in a possibly different form. At the least, the instructions should have the `--network none` and `--read-only` options, as the former in particular can cause issues that one doesn't have locally. I'd be interested to see if the new test runner works without network.

Reply:
Ah - I noticed the Ruby track didn't include anything, and it was broken for me as-written so I just ripped it out. You're right, it's worthwhile to actually fix it (done in the latest commit). I had to instruct Go to cache files to /tmp, and everything else worked just fine after fiddling with docker a bit.

Reply:
Excellent!